dotcom-rendering/src/web/components/MostViewedRightWrapper.importable.jsx

```diff
@@ -29,7 +34,8 @@ export const MostViewedRightWrapper = ({ limitItems, isAdFreeUser }: Props) => {
 	`;
 
 	useEffect(() => {
-		const checkHeight = (ref: RefObject<HTMLDivElement>) => {
+		/** @param {import('react').RefObject<HTMLDivElement | null>} ref */
+		const checkHeight = (ref) => {
```

Comment:
I find this particularly awkward to read in JSDOC, is there a nicer way to format it? At least a way to exclude needing `import('react')` from within the comment itself?

Reply:
I think `@typedef` can be used to import types at the top. I'll have a go.